posthog/celery.py

```diff
@@ -437,6 +448,7 @@ def update_cache_item_task(key: str, cache_type, payload: dict) -> None:
     from posthog.tasks.update_cache import update_cache_item
 
     update_cache_item(key, cache_type, payload)
+    get_client().delete(f"processing-{key}")
```

Comment:
Doesn't have to be done now but I feel like this could be abstracted along with the setter method so we don't end up with any change in one file that doesn't correctly update in the other?

Reply:
Oh, yep... [connascence of algorithm](https://connascence.io/algorithm.html) :)

Definitely didn't try and make this "right" yet